src/routes/console/project-[project]/settings/deleteProject.svelte

```diff
@@ -33,7 +33,12 @@
     };
 </script>
 
-<Modal bind:show={showDelete} onSubmit={handleDelete} warning>
+<Modal
+    bind:show={showDelete}
+    onSubmit={handleDelete}
+    icon="exclamation"
+    state="warning"
+    headerDivider={false}>
```

Comment:
seeing all those changes on modals, I think it makes sense to create a confirmation component